modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -91,138 +84,94 @@ This example covers a configuration with 1 custom Scope and 2 custom Collections
 }
 ----
 
-== Collections Configuration Description
-
-Here is the description for the custom `Scope` with 2 custom `Collections` configuration. 
-The database's structure, collections, sync functions, import filters, and index replicas are essential in accurately validating, filtering, and synchronizing documents within the database. 
-
-== Database Information
-
-=== db
-This sets the name of the database to "db".
-
-=== bucket
-This indicates the bucket's name (where the data will be stored), in this case, "bucket".
-
-== Scopes
-A Scope can contain a set of Collections. 
-You have the option to use the default provided by Couchbase or customize it to fit your specific needs.
-
-=== scope1
-You can define 1 custom scope per database with up to 1000 custom collections.
-
-If you don't specify a custom scope and collection, any documents you create will be saved in the default scope and collection.
-
-== Collections
-A Collection can contain a set of documents. 
-You have the option to use the default provided by Couchbase or customize it to fit your specific needs.
-
-=== Collection1
-Collection1 is storage for documents that are labelled as "default". 
-Its sync function guarantees that any changes made to documents in Collection1 are also reflected in a legacy system. 
-In addition, Collection1's import filter ensures that only documents labelled as "mobile" are imported.
+|===
```

Comment:
Moving this section to a table is 👍 

It's quite hard to read *formatted* as a table.
Instead, I'd recommend having each Field/Description formatted on new lines, with each pair separated by e new blank, e.g.:

```
| name                  
| The name of the database.

| bucket                
| The name of the bucket associated with the database.

...
```

Reply:
Also for the description, new sentence on new line.